packages/messaging/src/Message.ts

```diff
@@ -150,10 +150,12 @@ export function ensureOwnerIsSender({ body, sender }: IMessage): void {
       break
     case 'submit-document-stream':
       {
+	/*
```

Comment:
this is done as document would be taken out and we would bring VC as a native type inside of SDK.